qutip/solver/ode/evolver.py

```diff
@@ -0,0 +1,141 @@
+
+from ..evolver import Evolver, evolver_collection
+from .verner7efficient import vern7
+from .verner9efficient import vern9
+from .wrapper import QtOdeFuncWrapper
+import numpy as np
+from ...core import data as _data
+
+__all__ = ['EvolverVern', 'EvolverDiag']
+
+
+class EvolverVern(Evolver):
+    # -------------------------------------------------------------------------
+    # Solve an ODE for func.
+    # Calculate the required expectation values or invoke callback
+    # function at each time step.
+    #
+    # Use verner method implimented in cython
+    #
```

Comment:
This should be a multiline docstring imo since it is a public class: https://www.python.org/dev/peps/pep-0257/

```
A class to use the Verner method [] for solving ...
```